cookbook/agent_concepts/knowledge/vector_dbs/surrealdb/surreal_filter.py

```diff
@@ -0,0 +1,105 @@
+from agno.agent import Agent
+from agno.knowledge.pdf import PDFKnowledgeBase
```

Comment:
Thanks a lot for the push @dsaad68, can we please move this file to here- `cookbook/agent_concepts/knowledge/filters/filtering_surrealdb.py`

Then we good to merge!